homeassistant/components/adax/__init__.py

```diff
@@ -17,3 +17,18 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Unload a config entry."""
     return await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
+
+
+async def async_migrate_entry(hass, config_entry):
```

Comment:
Please add type hints

Reply:
We don't require type hints unless at least one arg is typed or strict typing is enabled.  Its up to the codeowner to opt-in.

Reply:
Of course adding them it defiantly preferred :)